core/block.ts

```diff
@@ -356,7 +357,6 @@ export class Block implements IASTNodeLocation, IDeletable {
       }
     } finally {
       eventUtils.enable();
-      this.disposed = true;
```

Comment:
Do you know where the exception this try/catch is wrapping is thrown? Do we still need this here in case it throws prior to disposed being set to true in the try?

Reply:
Ha no. Back in 2016 a contributor just wrapped everything between an `Events.disable()` and an `Events.enable()` in a `try...finally` to make sure that events were always re-enabled: https://github.com/google/blockly/pull/348